### ISSUE TITLE
Testing of the script 

### ISSUE THREAD

**Issue by meets2tarun** [fossasia/meilix#287]
- Issue type: Feature request 

We have implement a script to download the script folder. The script needs testing to download and run all the scripts present inside the folder.
Reference: https://github.com/fossasia/meilix/pull/284/files#diff-333f2f40be6e16a3a754ae82087c11fbR171

**Comment by meets2tarun** [comment]
Yes, now we can look into implementing the feature of wallpaper with the help of following strategy.

**Comment by pranav1698** [comment]
@tabesin @meets2tarun this can be closed, we are not downloading any folder from meilix-generator in the current build
